src/site/content/en/blog/script-component/index.md

```diff
@@ -0,0 +1,221 @@
+    </thead>
+    <tbody>
+        <tr>
+            <td>Google Tag Manager with async</td>
+            <td rowspan="2" >Script component with strategy = lazyonload for each of the four scripts
+            </td>
+        </tr>
+        <tr>
+            <td>Twitter Follow button with async
+            </td>
+        </tr>
+        <tr>
+            <td>YouTube Subscribe button without async or defer
+            </td>
+        </tr>
+        <tr>
+            <td>LinkedIn Follow button without async or defer
+            </td>
+        </tr>
+    </tbody>
+    <caption>Script and Script Component configuration for demo 2 with 4 scripts.</caption>
+</table>
+</div>
+
+{% Img src="image/IypihH3o5cSpEMVp5i08dp69otp2/BAWQVlyBX1UUJOMyrBJm.gif", alt="Video showing loading progress with and without the Script component ", width="800", height="306" %}
+
+As seen in the video, FCP occurs at 0.4s on the page without the Script component and 0.9 sec with the Script component. 
+
+
+## What’s next for the Script component
+
+While the strategy options for `afterInteractive` and `lazyOnload` provide significant control over render-blocking scripts, we are also exploring other options that would increase the utility of the Script component. 
+
+
+### Using web workers
+
+[Web workers](https://developer.mozilla.org/docs/Web/API/Web_Workers_API/Using_web_workers) can be used to run independent scripts on background threads which can free up the main thread to handle processing user interface tasks and improve performance. Web Workers are best suited for offloading JavaScript processing, rather than UI work, off the main thread. Scripts used for customer support or marketing, which typically do not interact with the UI, may be good candidates for execution on a background thread. A lightweight third-party library - [PartyTown](https://github.com/BuilderIO/partytown) may be used to isolate such scripts into a web worker.
+
+With the current implementation of the Next.js script component, we recommend deferring these scripts on the main thread by setting the strategy to afterInteractive or lazyOnload.In the future, we propose introducing a new strategy option, 'worker', which will allow Next.js to use PartyTown or a custom solution to run scripts on web workers. We welcome comments from developers on this [RFC](https://github.com/vercel/next.js/discussions/31517).
+
+
+### Minimizing CLS
+
+Third-party embeds like advertisements, video, or social media feed embeds can cause layout shifts when lazy-loaded. This affects the user experience and the [Cumulative Layout Shift (CLS)](/cls/) metric for the page. CLS can be minimized by specifying the size of the container where the embed will load. 
+
+The Script component may be used to load embeds that can cause CLS. We are considering augmenting it to provide configuration options that will help reduce the CLS. This could be made available within the Script component itself or as a companion component.    
```

Comment:
```suggestion
The Script component may be used to load embeds that can cause layout shifts. We are considering augmenting it to provide configuration options that will help reduce the CLS. This could be made available within the Script component itself or as a companion component.    
```